plugins/vpc-shared-eni/plugin/commands.go

```diff
@@ -130,18 +130,18 @@ func (plugin *Plugin) Del(args *cniSkel.CmdArgs) error {
 	log.Infof("Executing DEL with netconfig: %+v ContainerID:%v Netns:%v IfName:%v Args:%v.",
 		netConfig, args.ContainerID, args.Netns, args.IfName, args.Args)
 
-	// Find the ENI.
+	// Initialize a new ENI object
```

Comment:
Please squeeze your commits and remove these comment/text changes. They are not relevant.

Reply:
So, the comment say `Find the ENI` but we're actually initializing a new ENI object here. Thought it would be helpful to update the comment here since we're looking @ DEL flow in this PR.